### PR TITLE
fix(clerk-js): Respect color text

### PR DESCRIPTION
**Pull request by desiprisg**
## Description

This will result in incosistencies across components and design. We need to add more colors for it to be exactly the same. However, we need to use `colorText` and not `blackAlpha` for `color`, as it needs to be overridable by `appearance.variables.colorText`. 

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Comment by desiprisg** [comment]
I'm leaving this open until it can be discussed with design

File: packages/clerk-js/src/ui/hooks/usePopover.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ type UsePopoverProps = {
 export type UsePopoverReturn = ReturnType<typeof usePopover>;
 
 export const usePopover = (props: UsePopoverProps = {}) => {
-  const { bubbles = true, shoudFlip = true, outsidePress } = props;
+  const { bubbles = false, shoudFlip = true, outsidePress } = props;
```
Comment (panteliselef) [inline packages/clerk-js/src/ui/hooks/usePopover.ts:23]: ❓Is this related ?

Comment (desiprisg) [reply packages/clerk-js/src/ui/hooks/usePopover.ts:23]: No it's not I must have started off another branch. I'll make the necessary corrections.

**Comment by desiprisg** [comment]
!preview

**Comment by clerk-cookie** [comment]
Hey @desiprisg, your preview is currently deploying and should be available shortly.

| Status | Preview | Updated (UTC) |
| :-- | :-- | :-- |
| :cook: _Deploying..._ | [Inspect](https://javascript-preview-juldxe927.clerkstage.dev) | Jan 23, 2024 12:00 PM |

**Comment by clerk-cookie** [comment]
Hey @anagstef - the snapshot version command generated the following package versions:
| Package | Version |
| --- | --- |
| @clerk/backend | 1.0.1-snapshot.v9281f71 |
| @clerk/chrome-extension | 1.0.1-snapshot.v9281f71 |
| @clerk/clerk-js | 5.0.1-snapshot.v9281f71 |
| @clerk/elements | 0.0.3-snapshot.v9281f71 |
| @clerk/clerk-expo | 1.0.1-snapshot.v9281f71 |
| @clerk/fastify | 1.0.1-snapshot.v9281f71 |
| gatsby-plugin-clerk | 5.0.1-snapshot.v9281f71 |
| @clerk/localizations | 2.0.1-snapshot.v9281f71 |
| @clerk/nextjs | 5.0.1-snapshot.v9281f71 |
| @clerk/clerk-react | 5.0.1-snapshot.v9281f71 |
| @clerk/remix | 4.0.1-snapshot.v9281f71 |
| @clerk/clerk-sdk-node | 5.0.1-snapshot.v9281f71 |
| @clerk/shared | 2.0.1-snapshot.v9281f71 |
| @clerk/themes | 2.0.1-snapshot.v9281f71 |
| @clerk/types | 4.0.1-snapshot.v9281f71 |

Tip: Use the snippet copy button below to quickly install the required packages.
`@clerk/backend`
```sh
npm i @clerk/backend@1.0.1-snapshot.v9281f71 --save-exact
```
`@clerk/chrome-extension`
```sh
npm i @clerk/chrome-extension@1.0.1-snapshot.v9281f71 --save-exact
```
`@clerk/clerk-js`
```sh
npm i @clerk/clerk-js@5.0.1-snapshot.v9281f71 --save-exact
```
`@clerk/elements`
```sh
npm i @clerk/elements@0.0.3-snapshot.v9281f71 --save-exact
```
`@clerk/clerk-expo`
```sh
npm i @clerk/clerk-expo@1.0.1-snapshot.v9281f71 --save-exact
```
`@clerk/fastify`
```sh
npm i @clerk/fastify@1.0.1-snapshot.v9281f71 --save-exact
```
`gatsby-plugin-clerk`
```sh
npm i gatsby-plugin-clerk@5.0.1-snapshot.v9281f71 --save-exact
```
`@clerk/localizations`
```sh
npm i @clerk/localizations@2.0.1-snapshot.v9281f71 --save-exact
```
`@clerk/nextjs`
```sh
npm i @clerk/nextjs@5.0.1-snapshot.v9281f71 --save-exact
```
`@clerk/clerk-react`
```sh
npm i @clerk/clerk-react@5.0.1-snapshot.v9281f71 --save-exact
```
`@clerk/remix`
```sh
npm i @clerk/remix@4.0.1-snapshot.v9281f71 --save-exact
```
`@clerk/clerk-sdk-node`
```sh
npm i @clerk/clerk-sdk-node@5.0.1-snapshot.v9281f71 --save-exact
```
`@clerk/shared`
```sh
npm i @clerk/shared@2.0.1-snapshot.v9281f71 --save-exact
```
`@clerk/themes`
```sh
npm i @clerk/themes@2.0.1-snapshot.v9281f71 --save-exact
```
`@clerk/types`
```sh
npm i @clerk/types@4.0.1-snapshot.v9281f71 --save-exact
```

**Comment by desiprisg** [comment]
Closing in favour of #2680
